Homework1/MatrixMultiplication/MatrixMultiplication/TimeMeasurement.cs

```diff
@@ -0,0 +1,88 @@
+/// </summary>
+public class TimeMeasurement
+{
+    private readonly int testsCount;
+    private readonly double[] parallelTests;
+    private readonly double[] ordinaryTests;
+
+    public double ParallelExpectation { get; private set; }
+    public double OrdinaryExpectation { get; private set; }
+    public double ParallelDeviation { get; private set; }
+    public double OrdinaryDeviation { get; private set; }
+
+    public TimeMeasurement(int testsCount)
+    {
+        this.testsCount = testsCount;
+        parallelTests = new double[testsCount];
+        ordinaryTests = new double[testsCount];
+    }
+    
+    /// <summary>
+    /// Updates measurements
+    /// </summary>
+    public void UpdateTestInfo()
+    {
+        var timer = new Stopwatch();
+        var random = new Random();
+        const int commonDimension = 1000;
+        for (var i = 0; i < testsCount; i++)
+        {
+            var matrix1Lines = random.Next(1, 500);
+            var matrix2Columns = random.Next(1, 500);
+            var testMatrix1 = new int[matrix1Lines, commonDimension];
+            var testMatrix2 = new int[commonDimension, matrix2Columns];
+            FillMatrix(testMatrix1);
+            FillMatrix(testMatrix2);
+            var matrixMultiplier = new MatrixMultiplier(new Matrix(testMatrix1), new Matrix(testMatrix2));
+            timer.Restart();
+            matrixMultiplier.ParallelMultiply();
+            parallelTests[i] = timer.Elapsed.Seconds + timer.ElapsedMilliseconds/(double)1000;
+            timer.Restart();
+            matrixMultiplier.Multiply();
+            ordinaryTests[i] = timer.Elapsed.Seconds + timer.ElapsedMilliseconds/(double)1000;
+        }
+
+        OrdinaryExpectation = ordinaryTests.Sum();
+        ParallelExpectation = parallelTests.Sum();
+        OrdinaryDeviation = Math.Sqrt(ordinaryTests.Select(test => Math.Pow(test - OrdinaryExpectation, 2)).Sum());
+        ParallelDeviation = Math.Sqrt(parallelTests.Select(test => Math.Pow(test - ParallelExpectation, 2)).Sum());
+
+        void FillMatrix(int[,] matrix)
+        {
+            for (var i = 0; i < matrix.GetLength(0); i++)
+            {
+                for (var j = 0; j < matrix.GetLength(1); j++)
+                {
+                    matrix[i, j] = random.Next(10);
+                }
+            }
+        }
+    }
+
+    /// <summary>
+    /// Returns measurement strings
+    /// </summary>
+    public string[] GetMeasurements()
+    {
+        return new[]
+        {
+            $"Measurements for ordinary multiply: Expectation - {OrdinaryExpectation}, Deviation - {OrdinaryDeviation}",
+            $"Measurements for parallel multiply: Expectation - {ParallelExpectation}, Deviation - {ParallelDeviation}"
+        };
+    }
+
+    
+    /// <summary>
+    /// Writes measurement string in testinfo.txt file
+    /// </summary>
+    public void WriteInfo()
+    {
+        File.WriteAllLines("TestInfo.txt",GetMeasurements());
```

Comment:
```suggestion
        File.WriteAllLines("TestInfo.txt", GetMeasurements());
```
Ну и после запятой, конечно, тоже всегда пробел